public/creators-landing/src/locale/en.js

```diff
@@ -0,0 +1,102 @@
+export default {
+  nav: {
+    logo_href: '/',
+    signup: 'sign up',
+    signup_href: '/sign-up',
+    login: 'log in',
+    login_href: '/sign-in'
+  },
+  main: {
+    home: {
+      headline: 'Earn more for content you publish to the web',
+      subhead:
+        'Your viewers who use the Brave Browser may have contributed money to you while surfing the web through Brave Rewards. Simply sign up as a verified content creator on Brave Rewards to start collecting your contributions.',
+      examples: {
+        headline: 'Earn more through...',
+        website: 'Your own website',
+        publication: 'Your own publication',
+        youtube: 'YouTube',
+        Twitch: 'Twitch'
+      },
+      btn: {
+        signup: 'Sign up now'
+      }
+    },
+    signin: {
+      heading: 'Sign in to your account',
+      subhead:
+        'You will receive an email momentarily with a secure login link.',
+      input_placeholder: 'Enter your email',
```

Comment:
Usually javascript uses `camelCase` for variable names

Reply:
fixed